README.md

```diff
@@ -20,6 +20,7 @@ docker run
     -e API_KEY={API_KEY}
     -e HOST=api-dev.traveltimeapp.com //optional
     -e COUNTRY=gb //optional
+    -e COORDINATES=-8,13 // optional, this will overwrite COUNTRY env var
```

Comment:
Will it actually overwrite the COUNTRY env var? That var is also used to construct the service url, not just to pick the coordinates.

Reply:
Can you specify the exact location where COUNTRY var is used for that? As far as I can see, it's not even used in `generageBody`

Reply:
```
  const country = __ENV.COUNTRY || 'uk'
  ...
  const query = __ENV.QUERY || `api/v2/${countryCode(country)}/time-filter/fast/${transportation}`
  ...
  const countryCoords = protoCountries[country]
```

Reply:
ah nvm, I see, proto is a bit different, I'll take a look

Reply:
Tested it, it seems like the country that's specified in the url for proto requests makes no difference for the calculation. It's possible to use UK coords with any other country in the url, the result is still the same. I could mention this in README or comments if you think it's worth doing so